whyChooseUs.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;700&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="./assets/css/whyChooseUs.css" />
+  <title> Trending Projects</title>
+</head>
+
+<body>
+
+    <section>
+      <div class="whychooseus-page-main"></div>
+        <div class="whychooseus-page-outer">
+            <p class="whychooseus-page-heading">
+              Why Choose Us
+            </p>
+        </div>
+        <div class="whychooseus-card-container">
+          <div class="whychooseus-card">
+            <div class="whychooseus-page-img-div">
+              <img src="C:\Users\prash\OneDrive\Desktop\ReactJs Code\book-my-wing\assets\images\bestDeal1.png" />
+            </div>
+            <div class="row-container">
+              <div class="whychooseus-data-row1">
+                <p>Best Deals</p>
+              </div>
+              <div class="whychooseus-data-row2">
+               <p> With data-driven approach IndexTap helps in negotiation and ensures best property deal.</p>
+              </div>
+              
+            </div>
+    
+    
+          </div>
+          <div class="whychooseus-card">
+            <div class="whychooseus-page-img-div">
+              <img src="C:\Users\prash\OneDrive\Desktop\ReactJs Code\book-my-wing\assets\images\stackView.png" />
```

Comment:
@kajalwakchaure18 path asa nai deycha, keep all the images in our assets/images folder and path should be ../assets/images/stackView.png, change it everywhere

Reply:
made changes as per suggestion